### PR TITLE
Added screenshots and trailers to game details

### LINKED ISSUE #158: Add trailer and screenshots to game details

### PR DESCRIPTION
**Pull request by Mkdantas**
Added a slider with videos and screenshots of the game in game details.

This PR resolves #158 

hydra 2 captura
captura hydra

### CODE REVIEW

File: .env.example
Comment (zamitto) [inline .env.example:1]: Please, reverse the exclusion of this file

File: src/renderer/src/pages/game-details/gallery-slider.tsx
Comment (zamitto) [inline src/renderer/src/pages/game-details/gallery-slider.tsx:1]: please, format this file with prettier

File: src/renderer/src/pages/game-details/game-details.tsx [hunk 1]
```diff
@@ -87,7 +88,7 @@ export function GameDetails() {
   useEffect(() => {
     getGame();
   }, [getGame, gameDownloading?.id]);
-
+  console.log(gameDetails)
```
Comment (zamitto) [inline src/renderer/src/pages/game-details/game-details.tsx:91]: please, remove this console log

**Comment by zamitto** [comment]
I added some comments after a *quick* check

**Comment by Mkdantas** [comment]
> I added some comments after a **quick** check

Thank you, will resolve this issues and let you know when is ready

**Comment by zamitto** [comment]
also, the screenshots seems too big, maybe send in the discord channel how it is now so other people say what they think

**Comment by Mkdantas** [comment]
> I added some comments after a **quick** check

Done, thank you for the rapid feedback. Let me know if you have more.

Let me know if the screenshots and videos slider is placed in the desired locaiton or if i should move it to the bottom of details page.

Also, i realized that the description does not have a title indicating where it starts, so if you want i can remove the title i created for the slider section

File: .env.example [hunk 2]
```diff
@@ -2,4 +2,4 @@ MAIN_VITE_STEAMGRIDDB_API_KEY=YOUR_API_KEY
 MAIN_VITE_ONLINEFIX_USERNAME=YOUR_USERNAME
 MAIN_VITE_ONLINEFIX_PASSWORD=YOUR_PASSWORD
 MAIN_VITE_SENTRY_DSN=YOUR_SENTRY_DSN
-RENDERER_VITE_SENTRY_DSN=YOUR_SENTRY_DSN
```
Comment (hydralauncher) [inline .env.example:5]: Please remove your change from this file

**Review by hydralauncher — CHANGES_REQUESTED** [review]

**Review by hydralauncher — APPROVED** [review]
LGTM

**Pull request merged**